### PR TITLE
Document about packageOverrides (re issue #4505)

### PR DESCRIPTION
**Pull request by seppeljordan**
I also untabified the file and cleaned whitespaces.  Contact me if the PR does not satisfy the contribution guidlines, i'd be happy to know and implement apropriate changes.

### CODE REVIEW

File: doc/packageconfig.xml [hunk 1]
```diff
@@ -1,70 +1,104 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
-         xmlns:xlink="http://www.w3.org/1999/xlink"
-         xml:id="chap-packageconfig">
+	 xmlns:xlink="http://www.w3.org/1999/xlink"
+	 xml:id="chap-packageconfig">
 
 <title><filename>~/.nixpkgs/config.nix</filename>: global configuration</title>
 
+<section><title>Allow unfree software</title>
```
Comment (domenkozar) [inline doc/packageconfig.xml]: It's not about `unfree` software but just settings to the packages.

File: doc/packageconfig.xml [hunk 2]
```diff
@@ -1,70 +1,104 @@
...
 
     <para>
-        A complete list of licenses can be found in the file
-        <filename>lib/licenses.nix</filename> of the nix package tree.
+	A complete list of licenses can be found in the file
+	<filename>lib/licenses.nix</filename> of the nix package tree.
     </para>
+</section>
+
+<section><title>Overriding existing packages</title>
```
Comment (domenkozar) [inline doc/packageconfig.xml]: Title should mention the function name `packageOverrides`

**Comment by domenkozar** [comment]
This should mostly include how the function is defined and used, we have narrative documentation in http://nixos.org/nixos/manual/sec-package-management.html#sec-customising-packages

**Comment by seppeljordan** [comment]
Okay, it is quite unnarrative now and focusses on the function itself.

**Pull request merged**